sdk/core/core-tracing/package.json

```diff
@@ -39,6 +39,7 @@
     "dist/",
     "dist-esm/lib/",
     "src/",
```

Comment:
Should we remove src/ then as well?

Reply:
Probably yes, but I think we have another issue that is tracking moving from `lib` to `src` and I was worried we'd regress this bug in that PR if we did.